include/libtorrent/kademlia/node.hpp

```diff
@@ -234,6 +234,10 @@ class TORRENT_EXTRA_EXPORT node : boost::noncopyable
 	libtorrent::dht_settings const& m_settings;
 
 private:
+
+	void update_counters();
+
+private:
```

Comment:
Sometimes I prefer to separate private methods from private fields (styles...:). I will put this method at the end of the final private section